modules/system/classes/MailManager.php

```diff
@@ -93,21 +96,73 @@ public function addContentToMailer($message, $code, $data, $plainOnly = false)
             return false;
         }
 
-        $this->addContentToMailerInternal($message, $template, $data, $plainOnly);
+        $html = $text = $template;
+
+        if ($plainOnly) {
+            $html = null;
+        }
+        $this->addContentToMailerInternal($message, $html, $text, $data);
 
         return true;
     }
 
     /**
-     * Internal method used to share logic between `addRawContentToMailer` and `addContentToMailer`
+     * Restore proper behavior in-line with Laravel Mailer
+     * Replaces both `addContentToMailer` and `addRawContentToMailer`
      *
      * @param \Illuminate\Mail\Message $message
-     * @param string $template
+     * @param string|null $view
+     * @param string|null $plain
+     * @param string|null $raw
+     * @param array $data
+     * @return bool
+     */
+    public function addContent($message, $view, $plain, $raw, $data)
+    {
+        $html = $text = null;
+
+        if (isset($view)) {
```

Comment:
Shouldn't this be `!is_null()` instead of isset()?

Reply:
@LukeTowers What's the advantage of using !is_null() ? They're both doing pretty much the same thing.

Reply:
It's more semantically correct. I don't personally agree with `isset()` returning false for variables that are set to null. It's within the realm of possibilities that that behaviour could be changed in a future version of PHP, but mostly it's for semantics. @bennothommo any thoughts?

Reply:
I must say I agree with you, but it's used extensively in october code base and Laravel. I can change it if you prefer, I don't really mind.

Reply:
Yes please, ideally we make the change over for any other similar cases as we come across them.

Reply:
@LukeTowers @mjauvin I'm admittedly of the opposite position. I predominantly use `isset` because it also safely handles undefined variables - I don't believe `is_null` does the same. Note that when we use null-coalesce syntax (`??`), that's technically firing an `isset` too.

`null` traditionally means the *absence* of a value, which is why I agree that `isset` returns `false` for it.

Reply:
@bennothommo that is true for accessing array values or object properties, but for direct variable access, isser isn't necessary.

Reply:
@mjauvin How so? It's the same principle.

Reply:
@bennothommo in this case those variables will never not exist since they're required in the constructor

Reply:
@LukeTowers that's fair enough in this case, but I'm against changing it wholesale.